check/flynt

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
can this be a part of check/format-incremental? Otherwise it's yet another thing you need to run before submitting a PR

Reply:
Done.

Reply:
+1 to that suggestion

Reply:
Done.